pymc/tests/test_sampling_jax.py

```diff
@@ -145,15 +156,18 @@ def test_get_jaxified_logp():
         dict(log_likelihood=False),
     ],
 )
-def test_idata_kwargs(sampler, idata_kwargs):
+@pytest.mark.parametrize("postprocessing_backend", [None, "cpu"])
+def test_idata_kwargs(sampler, idata_kwargs, postprocessing_backend):
     with pm.Model() as m:
         x = pm.Normal("x")
+        z = pm.Normal("z")
```

Comment:
Does the new variable add anything?

Reply:
While refactoring I had a false positive test without it. Then I refactored back but think this use case is why not